Installers/default.build

```diff
@@ -143,19 +143,20 @@ fi
     <!-- Linux Installer -->
     <if test="${(os == 'Unix' or os == 'MacOS') and directory::exists('../Tools/Pipeline/bin/Linux/AnyCPU/Release')}">
       
+      <!-- .deb installer -->
       <delete dir="Linux/tmp_deb" failonerror="false"/>
       <mkdir dir="Linux/tmp_deb"/>
       <copy todir="Linux/tmp_deb/DEBIAN/">
         <fileset basedir="Linux/DEBIAN"/>
       </copy>
-      <copy todir="Linux/tmp_deb/opt/monogame-pipeline">
+      <copy todir="Linux/tmp_deb/usr/lib/mono/xbuild/MonoGame/v3.0/Tools">
```

Comment:
I like the idea, but I can't do that because .deb installer wouldn't work (in order to install the new version, the previous version gets autouninstalled with it).

I could still do it for the .run installer, but then .run and .deb installer would be doing different things, and I wouldn't want to make a feature that is exclusive to one installer.